src/sentry/integrations/slack/utils/channel.py

```diff
@@ -17,11 +23,20 @@
 
 from . import logger
 
+SLACK_GET_CHANNEL_ID_PAGE_SIZE = 200
 SLACK_DEFAULT_TIMEOUT = 10
 MEMBER_PREFIX = "@"
 CHANNEL_PREFIX = "#"
 strip_channel_chars = "".join([MEMBER_PREFIX, CHANNEL_PREFIX])
 
+
+@dataclass
```

Comment:
should this be frozen?

Reply:
It can't be because i modify the instance in the function.